NOTICE

```diff
@@ -1,3 +1,4 @@
 Kubernetes Anywhere
 Copyright 2015-2016 Weaveworks Ltd.
```

Comment:
Ok, that sounds good. I basically have no idea what I'm doing so I'm going to have someone sign off and the this LGTM.